example/accel/CMakeLists.txt

```diff
@@ -68,3 +78,12 @@ if(Geant4_VERSION VERSION_LESS 11.1)
   )
 endif()
 
+if(Geant4_VERSION VERSION_GREATER_EQUAL 11.0)
+  add_example(trackingmanager-offload)
+  set_tests_properties(trackingmanager-offload PROPERTIES
+    ENVIRONMENT "${_test_env}"
+    LABELS "app"
+  )
+endif()
```

Comment:
I wonder with the more complex threading going on, should we run these examples with serial/mt/task? (Task is the default in 11.)

Reply:
Sorry, realised my comment here was a bit vague - I'll go ahead and add these tests unless you say otherwise.

Reply:
Now done - it should work all the way back to Geant4 10.6, worst case 10.7.

Reply:
Nice!! Great work, thanks @drbenmorgan !